sonar/modules/deposits/jsonschemas/deposits/deposit-v1.0.0_src.json

```diff
@@ -0,0 +1,485 @@
+                "minLength": 1
+              },
+              "url": {
+                "title": "URL",
+                "type": "string",
+                "minLength": 1,
+                "pattern": "^https?://.+"
+              }
+            }
+          }
+        },
+        "specificCollections": {
+          "title": "Specific collections",
+          "description": "The names of the institution's specific/patrimonial collections to which this document belongs",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          }
+        },
+        "classification": {
+          "$ref": "classification-v1.0.0.json"
+        },
+        "abstracts": {
+          "title": "Abstracts",
+          "type": "array",
+          "items": {
+            "title": "Abstract",
+            "type": "object",
+            "required": [
+              "language",
+              "abstract"
+            ],
+            "additionalProperties": false,
+            "propertiesOrder": ["language", "abstract"],
+            "properties": {
+              "language": {
+                "$ref": "language-v1.0.0.json"
+              },
+              "abstract": {
+                "title": "Abstract",
+                "type": "string",
+                "minLength": 1,
+                "template": {
+                  "type": "textarea",
+                  "rows": 3
+                }
+              }
+            }
+          }
+        },
+        "subjects": {
+          "title": "Subjects",
+          "type": "array",
+          "items": {
+            "title": "Subject",
+            "type": "object",
+            "required": [
+              "language",
+              "subjects"
+            ],
+            "additionalProperties": false,
+            "propertiesOrder": ["language", "subjects"],
+            "properties": {
+              "language": {
+                "$ref": "language-v1.0.0.json"
+              },
+              "subjects": {
+                "title": "Subjects",
+                "type": "array",
+                "minItems": 1,
+                "default": [],
+                "items": {
+                  "type": "string",
+                  "minLength": 1
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "contributors": {
+      "title": "Contributors",
+      "type": "array",
+      "minItems": 1,
+      "default": [
+        {}
+      ],
+      "items": {
+        "title": "Contributor",
+        "type": "object",
+        "required": [
+          "name",
+          "affiliation"
+        ],
+        "propertiesOrder": [
+          "name",
+          "affiliation"
+        ],
+        "properties": {
+          "name": {
+            "title": "Name",
+            "description": "Last name, first name, ex: Doe, John",
+            "type": "string",
+            "minLength": 1,
+            "pattern": "^[^,]+, [^,]+$"
+          },
```

Comment:
Some publication are special authors named "authors association" (very usual with Physics papers - see https://dial.uclouvain.be/pr/boreal/search/site/?f[0]=sm_creator:%22CMS%20Collaboration%22). 
These author doesn't have any affiliations and any coma into name.

Reply:
I'm sure you're right, but it's a PO consideration and I will discuss with them later.